src/CreativeCollab.css

```diff
@@ -6,11 +6,8 @@
 }
 
 .story-container {
-  position: absolute;
   border: 1px solid black;
-  padding-left: 1%;
-  bottom: 5%;
-  left: 20%;
-  width: 800px;
-  height: 380px;
+  height: 340px;
+  overflow: scroll;
```

Comment:
Nice work! We might want to use `overflow-y: auto` instead as it will automatically show the vertical scroller if the text exceeds the height of the `<div>`:

![overflow_example](https://user-images.githubusercontent.com/13009507/48308102-23fd7380-e52a-11e8-87f1-7e8df8b32546.gif)

You'll also notice that the message wraps to the next line and doesn't show a horizontal scroll bar - let's also add `overflow-wrap: break-word;` which does just that!